uo_sys_framework/GearModule/cfgFunctions/Olsen/fn_addItemRandomOrg.sqf

```diff
@@ -17,7 +17,12 @@
 #include "\x\UO_FW\addons\Main\script_macros.hpp"
 EXEC_CHECK(ALL);
 
-params ["_unit", "_loadoutType", "_items", ["_amount",1,[1]], ["_position", "NONE", [""]]];
+private ["_unit", "_loadoutType", "_items", "_amount", "_position", "_randomPick"];
```

Comment:
You can still use params command, just split it by each level of select/array nesting
so `params ["_unitArray", "_items", ["_amount",1,[1]], ["_position", "NONE", [""]]];
_unitArray params ["_unit","_loadoutType"];`

Reply:
`params` defaults to _this but you can use any array with it, like `_x` or any created var that points to an array.

Reply:
Like in latest push?

Reply:
need to remove the private line as params automatically does this for vars